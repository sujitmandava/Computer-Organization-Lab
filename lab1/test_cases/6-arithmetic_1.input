	.data
a:
	1
d:
	3
n:
	7
