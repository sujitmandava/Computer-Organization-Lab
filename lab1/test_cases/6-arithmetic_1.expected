Main Memory Contents:

65529		: 19
65530		: 16
65531		: 13
65532		: 10
65533		: 7
65534		: 4
65535		: 1
